api/api.go

```diff
@@ -91,10 +91,10 @@ func Setup(
 
 	// routes
 	api.Router.Handle("/filters", assert.DatasetType(http.HandlerFunc(api.postFilterBlueprintHandler))).Methods("POST")
-	api.Router.Handle("/filters/{filter_blueprint_id}", assert.FilterType(http.HandlerFunc(api.getFilterBlueprintHandler))).Methods("GET")
-	api.Router.Handle("/filters/{filter_blueprint_id}/dimensions", assert.FilterType(http.HandlerFunc(api.getFilterBlueprintDimensionsHandler))).Methods("GET")
+	api.Router.Handle("/filters/{filter_blueprint_id}", assert.DatasetTypeFromRoute(assert.FilterType(http.HandlerFunc(api.getFilterBlueprintHandler)))).Methods("GET")
```

Comment:
Not sure how you can assert the dataset type from a filter ID?

Reply:
As per offline conversations surrounding the ticket, the question has been addressed with the logic changed.